hack/redis/README.md

```diff
@@ -0,0 +1,22 @@
+# Redis
+
+Scripts and helpers for working with Redis
+
+## Load Testing Redis AWS (Elasticache)
+The following is taking from (redis-load)[https://github.com/ciaranRoche/redis-load] which generates random entries to redis
+### Usage 
+#### Prerequisites
+```
+export ns=<<namespace you want to deploy the load generator>>
+```
+#### Run Load Generator
+``` 
+oc create -f redis-load.yaml -n $ns
```

Comment:
```suggestion
oc create -f redis-load-test.yaml -n $ns
```

Reply:
Actually, maybe we could avoid creating the template in OpenShift and do

```
export cr=<<name of redis cr>>

oc process -f redis-load-test.yaml -p HOST=$(oc get secret -n $ns $(oc get redis example-redis -n $ns -o jsonpath='{.status.secretRef.name}') -o jsonpath='{.data.uri}' | base64 --decode) -p PORT=6379 -p ENTRIES=<<no of entries>> -p USERS=<<no of concurrent users>> | oc create -f -
```

? or something like that, then we don't need to create and delete it